repos/system_upgrade/common/actors/opensshconfigscanner/tests/test_readopensshconfig_opensshconfigscanner.py

```diff
@@ -351,6 +351,18 @@ def fake_producer(*args):
     assert cfg.subsystem_sftp == 'internal-sftp'
 
 
+@pytest.mark.parametrize('config_line,option_name,invalid_value', [
+    ('PermitRootLogin NO', 'PermitRootLogin', 'NO'),
+    ('UsePrivilegeSeparation YES', 'UsePrivilegeSeparation', 'YES'),
+])
+def test_parse_config_invalid_option_case(config_line, option_name, invalid_value):
+    config = [config_line]
+
+    with pytest.raises(StopActorExecutionError) as err:
+        parse_config(config)
+
+    assert 'Invalid value for SSH configuration option' in str(err.value)
```

Comment:
and in such a case we could still check the prefix:

```suggestion
    assert err.value.startswith('Invalid SSH configuration')
```